server/embedded/src/main/resources/log4j2.xml

```diff
@@ -1,5 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 
+<!--
+When making edits to this file, please communicate with systems engineering and
+account managers so that we can coordinate edits with other customized copies of the file
```

Comment:
We also need to keep this in sync with the version in `gradlePlugins`, so perhaps make note of that?